analytics/src/views/Dashboard/components/Map/Map.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
```

Comment:
If we are taking the approach of "features first" under the "views" directory, might we consider "Map" to be a feature of its own?